provision/acc_provision/flavors.yaml

```diff
@@ -108,6 +108,24 @@ flavors:
     status: null
     hidden: false
     order: 1
+  openshift-3.10:
+    desc: Red Hat OpenShift Container Platform 3.10
+    default_version: 1.9
```

Comment:
I know we have been setting this to 1.9 earlier, but is this correct for OpenShift?